src/resqml2_0_1/RockFluidOrganizationInterpretation.h

```diff
@@ -0,0 +1,93 @@
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-----------------------------------------------------------------------*/
+#pragma once
+
+#include "resqml2/AbstractGridRepresentation.h"
+#include "resqml2_0_1/AbstractOrganizationInterpretation.h"
+#include "resqml2_0_1/EarthModelInterpretation.h"
+#include "resqml2_0_1/OrganizationFeature.h"
+
+namespace RESQML2_0_1_NS
+{
+	/**
+	* This class is a container for other organizations that are consistent to each others.
+	*/
+	class DLL_IMPORT_OR_EXPORT RockFluidOrganizationInterpretation : public AbstractOrganizationInterpretation
+	{
+	public:
+		/**
+		* Only to be used in partial transfer context
+		*/
+		RockFluidOrganizationInterpretation(gsoap_resqml2_0_1::eml20__DataObjectReference* partialObject) : AbstractOrganizationInterpretation(partialObject) {}
+
+		/**
+		* Creates an instance of this class in a gsoap context.
+		* @param orgFeat			The feature the instance interprets. It must be a stratigraphic organization feature.
+		* @param guid				The guid to set to the interpretation. If empty then a new guid will be generated.
+		* @param title				A title for the instance to create.
+		* @param rank				The rank index of this interpretation within a stratigraphic column
+		* @param orderingCriteria	How the included horizons are ordered.
+		*/
+		RockFluidOrganizationInterpretation(class OrganizationFeature * orgFeat, const std::string & guid, const std::string & title, ULONG64 index);
+
+		/**
+		* Creates an instance of this class by wrapping a gsoap instance.
+		*/
+		RockFluidOrganizationInterpretation(gsoap_resqml2_0_1::_resqml2__RockFluidOrganizationInterpretation* fromGsoap) : AbstractOrganizationInterpretation(fromGsoap) {}
+
+		/**
+		* Destructor does nothing since the memory is managed by the gsoap context.
+		*/
+		~RockFluidOrganizationInterpretation() {}
+		
+		/**
+		* @return The count of grid representation assocaited to this strati organization.
+		*/
+		unsigned int getGridRepresentationCount() const;
+
+		/**
+		* Get a grid representation associated to this strati org interp by means of its index.
+		* @param index	The index of the grid representation to get in the array of grid representaitons of this strati org interp.
+		*/
+		RESQML2_NS::AbstractGridRepresentation* getGridRepresentation(const unsigned int & index) const;
+
+		/**
+		* Check if a grid representation is wether associated to this strati org interp or not.
+		* @param gridRep	The grid representation to check its assocaition with this strati org interp.
+		* @return			True or false.
+		*/
+		bool isAssociatedToGridRepresentation(RESQML2_NS::AbstractGridRepresentation* gridRep) const;
+
+		static const char* XML_TAG;
+		virtual std::string getXmlTag() const {return XML_TAG;}
+
+		ULONG64 getRockFluidUnitInterpretationIndex() const;
```

Comment:
Should be deleted. It has no meaning. It is a bug in the datamodel : http://docs.energistics.org/#RESQML/RESQML_TOPICS/RESQML-500-106-0-R-sv2010.html

Reply:
I confirm it must be deleted.
It is easier to delete and it is sure that we would not need it longer term (For example RESQML2.2 has not even no more this attribute).

Reply:
still unresolved

Reply:
Actually it was resolved in commit 57b953a